### ISSUE TITLE
No Rest Api, No Connection

### ISSUE THREAD

**Issue by JakkitDE** [WooMinecraft/WooMinecraft#290]
Newest WordPress, WooCommerce and WooMinecraft Versions on Java 1.19 etc. Connection failure.. And no rest api registration.
So I don't know what to do. I have already tried everything. Different MC-Hosts, Different Website Hosts... No Chance.

**Comment by JayWood** [comment]
Check your REST url `/wp-json/wp/v1/posts` - if this fails, your REST API is not available. 

> no rest api registration

What does this mean?

**Comment by JakkitDE** [comment]
> Check your REST url `/wp-json/wp/v1/posts` - if this fails, your REST API is not available.

code | "rest_no_route"
is the result. On my different hosts.. How to fix this?

**Comment by JayWood** [comment]
Do you even have a `/wp-json/` route? Are you using WordPress? What's the WordPress url?

**Comment by JakkitDE** [comment]
> Do you even have a `/wp-json/` route? Are you using WordPress? What's the WordPress url?

Yea, I think that seems right. https://golden-sky.de/wp-json
only the route doesnt exist as far as I understand.

**Comment by JayWood** [comment]
Yea looking at the wp-json feed the /wmc/ route does not exist. You have the WooMinecraft WordPress plugin installed on the WP side too right?

**Comment by JakkitDE** [comment]
> Yea looking at the wp-json feed the /wmc/ route does not exist. You have the WooMinecraft WordPress plugin installed on the WP side too right?

Of course. 2 years before I have used WooMinecraft in combination with woocommerce for a long time and made many sales successfully. Its easy to setup. But now it doesnt work for me.
